include/AMOCommand.hpp

```diff
@@ -21,7 +21,8 @@ class AMOCommand {
   AMOCommand(const AMOCommand &command);
   ~AMOCommand();
 
-  bool operator< (const AMOCommand &r) const;
+  bool operator<(const AMOCommand &r) const;
```

Comment:
nit: add space between operator and first paren: `bool operator< (const AMOCommand &r) const;`